lib/osf-components/addon/components/registries/finalize-registration-modal/component.ts

```diff
@@ -38,6 +41,18 @@ export default class FinalizeRegisrationModalComponent extends Component {
         }
     }
 
+    @computed('manager.draftManager.moderatedProvider')
+    get noticeText() {
+        const translationOptions = { learnMoreLink: this.learnMoreLink, htmlSafe: true };
+        if (this.manager.draftManager.moderatedProvider) {
```

Comment:
Nitpick: `moderatedProvider` here looks like a `RegistrationProvider` instance instead of a string. If it is merely an alias of the provider's `reviewsWorkflow`, might as well just call it that.